### ISSUE TITLE
Use data from Atom for the math toolbox

### ISSUE THREAD

**Issue by M4xi1m3** [Omega-Numworks/Omega#129]
Allows to free up some memory by removing duplicate data.

**Comment by MixedMatched** [comment]
This contradicts #130 though, doesn't it? If you try to build without Atom, toolbox would fail.

**Comment by quentinguidee** [comment]
Yes, I think we have to do the opposite (so remove the data in Omega-Atom)
